src/dotnet/Users.UI.Blazor/Components/OwnAccountEditorModal.razor

```diff
@@ -102,7 +149,7 @@
         _account = await Accounts.GetOwn(Session, default);
         _form.Name = _account.Name;
         _form.LastName = _account.LastName;
-        _form.Email = _account.Email;
+        _form.Email = _account.Email.IsNullOrEmpty() ? null : _account.Email;
```

Comment:
There is another extension usefull in this case: _account.Email.NullIfEmpty()

Reply:
done